evm/gateway.js

```diff
@@ -55,6 +56,17 @@ const getSignedWeightedExecuteInput = async (data, operators, weights, threshold
     );
 };
 
+const fetchBatchData = async (apiUrl, batchId) => {
+    try {
+        const response = await httpGet(`${apiUrl}/${batchId}`);
+        const data = response?.execute_data;
```

Comment:
also check the status of the signing session in the output. If signing hasn't completed, then the execute data value won't work